octoprint_v8theme/static/js/v8theme.js

```diff
@@ -9,6 +9,8 @@ $(function() {
             $("#control_link").addClass("active");
             $("#control").prepend($("#temp").contents());
             $("#control").addClass("active");
+            $("#temperature-graph").closest(".row").removeAttr("style");
+            $("#temperature-graph").closest(".row").attr("class", "row-fluid");
```

Comment:
gotcha. yeah I knew they existed but unaware of the data structure difference. I just wanted to change the reference of the row class and knew that element only had one class.